<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:		#66545e;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="From_File">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: 	#aa6f73 ;
color: 	#f6e0b5 ;</string>
    </property>
    <property name="text">
     <string>Choose From Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font:bold 18pt &quot;Times New Roman&quot;;
color: 		#f6e0b5 ;</string>
    </property>
    <property name="text">
     <string>EASY TRANSLATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Take_Pic">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: 	#f6e0b5 ;
background-color: 			#aa6f73 ;</string>
    </property>
    <property name="text">
     <string>Take Picture</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
